developer_manual/digging_deeper/reference.rst

```diff
@@ -0,0 +1,604 @@
+              "link": "https://github.com"
+            },
+            "openGraphObject": {
+              "id": "https://github.com",
+              "name": "GitHub: Let’s build from here",
+              "description": "GitHub is where over 100 million developers shape the future of software, together. Contribute to the open source community, manage your Git repositories, review code like a pro, track bugs and fea...",
+              "thumb": "https://my.nextcloud.org/core/references/preview/3097fca9b1ec8942c4305e550ef1b50a",
+              "link": "https://github.com"
+            },
+            "accessible": true
+          }
+        }
+      }
+    }
+
+The link might be supported by a reference provider that also provides more information in a rich object.
+The generic openGraphObject is still returned. It contains a title, description and image URL if the matching reference
+provider defined those correctly.
+For example, resolving ``https://www.themoviedb.org/movie/70981`` if the ``integration_tmdb`` app is installed will return:
+
+.. code-block:: json
+
+    "data": {
+      "references": {
+        "https://www.themoviedb.org/movie/70981": {
+          "richObjectType": "integration_tmdb_movie",
+          "richObject": {
+            "adult": false,
+            "budget": 130000000,
+            "genres": [
+              {
+                "id": 878,
+                "name": "Science Fiction"
+              },
+              {
+                "id": 12,
+                "name": "Adventure"
+              },
+              {
+                "id": 9648,
+                "name": "Mystery"
+              }
+            ],
+            "homepage": "https://www.20thcenturystudios.com/movies/prometheus",
+            "id": 70981,
+            "imdb_id": "tt1446714",
+            "original_language": "en",
+            "original_title": "Prometheus",
+            "overview": "A team of explorers discover a clue to the origins of mankind on Earth, leading them on a journey to the darkest corners of the universe. There, they must fight a terrifying battle to save the future of the human race.",
+            "popularity": 68.389,
+            "release_date": "2012-05-30",
+            "revenue": 403354469,
+            "runtime": 124,
+          },
+          "openGraphObject": {
+            "id": "https://www.themoviedb.org/movie/70981",
+            "name": "Prometheus",
+            "description": "30 mai 2012 - A team of explorers discover a clue to the origins of mankind on Earth, leading them on a journey to the darkest corners of the universe. There, they must fight a terrifying battle to save the future of the human race.",
+            "thumb": "https://my.nextcloud.org/apps/integration_tmdb/t/p/w500/qsYQflQhOuhDpQ0W2aOcwqgDAeI.jpg?fallbackName=???",
+            "link": "https://www.themoviedb.org/movie/70981"
+          },
+          "accessible": true
+        }
+      }
+    }
+
+Use the Smart Picker in your app
```

Comment:
the code block is still active here. add at least one more blank linkes above this line

Reply:
It was rendered correctly, still added a blank line, why not.